code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -705,6 +705,29 @@ static HookFunction hookFunction([]()
 			}
 		});
 
+        fx::ScriptEngine::RegisterNativeHandler("MUMBLE_REMOVE_VOICE_TARGET_CHANNEL", [](fx::ScriptContext& context)
+        {
+            auto id = context.GetArgument<int>(0);
+            auto channel = context.GetArgument<int>(1);
+
+            if (id >= 0 && id < 31)
+            {
+                auto targetChannel = fmt::sprintf("Game Channel %d", channel);
+                std::vector<VoiceTargetConfig::Target>& targets = vtConfigs[id].targets;
+                for (size_t i = targets.size(); i--;)
+                {
+                    VoiceTargetConfig::Target& target = targets[i];
+                    if (!target.channel.empty() && target.channel.compare(targetChannel) == 0)
+                    {
+                        targets.erase(targets.begin() + i);
```

Comment:
std::vector removal is nasty (rather use std::list as a replacement, maybe? especially if these can get big) and this is not a very idiomatic way to do removal (erase should remove another iterator, worst-case)

Reply:
I'll switch this to std::list as this has the possibility to add/remove players constantly, and depending on other peoples use cases it could be quite a bit larger than just 128 players